packages/yoastseo/spec/languageProcessing/helpers/highlighting/getMarkingsInSentenceSpec.js

```diff
@@ -0,0 +1,87 @@
+import getMarkingsInSentence from "../../../../src/languageProcessing/helpers/highlighting/getMarkingsInSentence";
```

Comment:
The unit test coverage for this file is not at 100%, have you tried to make it 100%?

Reply:
Made 100%! :)